autotyping/autotyping.py

```diff
@@ -657,3 +688,127 @@ def get_decorator_kind(dec: libcst.Decorator) -> Optional[DecoratorKind]:
         ):
             return DecoratorKind.abstractmethod
     return None
+
+
+# strategy taken from https://github.com/Zac-HD/hypothesis/blob/07ff885edaa0c11f480a8639a75101c6fe14844f/hypothesis-python/src/hypothesis/extra/ghostwriter.py#L319
+# but with more stuff moved up into the _NAMES tuples, since we don't care about ranges
+# and similar
+# fmt: off
+BOOL_NAMES = {
+    "keepdims", "verbose", "debug", "force", "train", "training", "trainable", "bias",
+    "shuffle", "show", "load", "pretrained", "save", "overwrite", "normalize",
+    "reverse", "success", "enabled", "strict", "copy", "quiet", "required", "inplace",
+    "recursive", "enable", "active", "create", "validate", "refresh", "use_bias",
+}
+INTEGER_NAMES = {
+    "width", "size", "length", "limit", "idx", "stride", "epoch", "epochs", "depth",
+    "pid", "steps", "iteration", "iterations", "vocab_size", "ttl", "count",
+    "offset", "seed", "dim", "total", "priority", "port",
+    "number", "num", "int",
+    }
+FLOAT_NAMES = {
+    "real", "imag", "alpha", "theta", "beta", "sigma", "gamma", "angle", "reward",
+    "tau", "temperature",
+    "learning_rate", "dropout", "dropout_rate", "epsilon", "eps", "prob",
+    "lat", "latitude",
+    "lon", "longitude",
+    "radius", "tol", "tolerance", "rate",
+    "treshold", "float",
```

Comment:
```suggestion
    "threshold", "float",
```

(not visible: me, sweating, checking whether I typo'd this into Hypothesis.  Thankfully no)

Reply:
Maybe the typo is common enough that it should also be in the list :)

Reply:
I checked https://github.com/HypothesisWorks/hypothesis/issues/3311, and nope, it's not 😜

Reply:
and a better solution would be to have a spell checker running in your IDE :sweat_smile: Misspelled function parameters are awful when you try to call them and specifying keyword arguments. (I'm totally not speaking from experience as a non-native that sometimes misspells hehe).
Not fixing it also gives the author an extra chance at reading the argument and noticing the typo.